### PR TITLE
Add about screen with contact info

### LINKED ISSUE #80: no about us information in the app

### PR DESCRIPTION
**Pull request by Idadelveloper**
This fixes #80 and #81 by providing an about screen with details about the app, contact information, and social links.

### CODE REVIEW

File: app/src/main/java/com/example/sw0b_001/AboutActivity.kt [hunk 1]
```diff
@@ -48,17 +49,32 @@ class AboutActivity : AppCompatActivity() {
...
         tutorialButton.setOnClickListener {
             openSocialLink("https://docs.smswithoutborders.com/docs/App%20Tutorial/New-Tutorial")
         }
+
+        val (appName, versionName) = try {
+            val packageInfo = packageManager.getPackageInfo(packageName, 0)
+            Pair(getString(packageInfo.applicationInfo.labelRes), packageInfo.versionName)
+        } catch (e: PackageManager.NameNotFoundException) {
+            Pair("RelaySMS", "")
+        }
```
Comment (sherlockwisdom) [inline app/src/main/java/com/example/sw0b_001/AboutActivity.kt:62]: Not needed

File: app/src/main/java/com/example/sw0b_001/AboutActivity.kt [hunk 2]
```diff
@@ -48,17 +49,32 @@ class AboutActivity : AppCompatActivity() {
...
+
+        val appTitleTextView = findViewById<TextView>(R.id.app_title)
+        appTitleTextView.text = appName
```
Comment (sherlockwisdom) [inline app/src/main/java/com/example/sw0b_001/AboutActivity.kt:65]: appTitleTextView.text = getString(R.string.app_name)

File: app/src/main/java/com/example/sw0b_001/AboutActivity.kt [hunk 3]
```diff
@@ -48,17 +49,32 @@ class AboutActivity : AppCompatActivity() {
...
+
+        val appVersionTextView = findViewById<TextView>(R.id.app_version)
+        appVersionTextView.text = versionName
```
Comment (sherlockwisdom) [inline app/src/main/java/com/example/sw0b_001/AboutActivity.kt:68]: appVersionTextView.text = BuildConfig.VERSION_NAME

**Review by sherlockwisdom — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/sw0b_001/AboutActivity.kt [hunk 4]
```diff
@@ -75,6 +75,15 @@ class AboutActivity : AppCompatActivity() {
         val currentYear = Calendar.getInstance().get(Calendar.YEAR)
         val copyrightTextView = findViewById<TextView>(R.id.copyright_text)
         copyrightTextView.text = "© $currentYear SMSWithoutBorders"
+
+        val shareButton = findViewById<Button>(R.id.share_button)
+        shareButton.setOnClickListener {
```
Comment (sherlockwisdom) [inline app/src/main/java/com/example/sw0b_001/AboutActivity.kt:75]: Move string to values/strings.xml

**Review by sherlockwisdom — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/email_icon.xml
[the remark this answers is not in the document]
Comment (Idadelveloper) [reply app/src/main/res/drawable/email_icon.xml:1]: @sherlockwisdom the background is already dark so the icons will be invisible. Maybe this md_theme_onPrimary?

File: app/src/main/java/com/example/sw0b_001/AboutActivity.kt (hunk 4 above)
Comment (Idadelveloper) [reply app/src/main/java/com/example/sw0b_001/AboutActivity.kt:67]: @sherlockwisdom check it out now

**Pull request merged**